pipenv/cli.py

```diff
@@ -2242,6 +2242,16 @@ def inline_activate_virtualenv():
 @click.option('--python', default=False, nargs=1, help="Specify which version of Python virtualenv should use.")
 def run(command, args, three=None, python=False):
     # Ensure that virtualenv is available.
+    if not project.virtualenv_exists and not PIPENV_USE_SYSTEM:
+        click.echo(
+            crayons.red(
+                u'This project has no virtualenv yet! Use $ pipenv install, for example, to create one.',
+                bold=True
+            ),
+            err=True
+        )
+        sys.exit(1)
```

Comment:
any chance an exception such as `PipenvMissingVirtualenv` could be thrown so it could be caught and handled?

Reply:
Probably not since this is a CLI application, I'm guessing we will want to stick to exit codes

Reply:
`$ pipenv install` should be the same color as literally every other shell example we spit out.

Reply:
@cs01 absolutely not.

Reply:
Ah yeah that makes sense. I only asked because I want to be able to programatically tell if I can skip running `pipenv install` or not. [This](https://github.com/kennethreitz/pipenv/issues/1080#issuecomment-345867519) would take care of that too.